ci/appveyor/win-set-cmake_generator.bat

```diff
@@ -0,0 +1,24 @@
+@echo off
+
+:set_generator_part
+if /i "%1" == "msvc15" (
+  set CMAKE_GENERATOR_PART=Visual Studio 15 2017

```

Comment:
you're using "non-parallel" build configuration - it is slowed than it could be
instead, I suggest using "NMake Makefiles JOM"
it requires adding C:\Qt\Tools\QtCreator\bin to PATH, but allows using:
`- cmake --build . --config %CONFIGURATION% -- -j5`
to parallel build in 5 threads